index.html

```diff
@@ -51,5 +51,154 @@ <h1>
+          <li class="back-end">Back End Dev</li>
+          <li class="year">2015</li>
+        </ul>
+        <p>A daily selection of privately 
+          personalized reads; no accounts 
+          or sign-ups required.</p>
+          <ul class="tag">
+            <li class="html">html</li>
+            <li class="cascade">css</li>
+            <li class="javascript">javascript</li>
+          </ul>
+          <div class="see-project-one">
+           <button class="see-project">See project</button>
+           </div>
+
+      </section>
+
+      <section class="card-Work-container">
+        <div class="snapshot-porfolio">
+          <img src="./images/snap 3.png" alt="">
+        </div>
+        <h2>Tonic</h2>
+        <ul class="tonic">
+          <li class="canopy">CANOPY</li>
+          <li class="back-end">Back End Dev</li>
+          <li class="year">2015</li>
+        </ul>
+      
+        <div>
+        <p>A daily selection of privately 
+          personalized reads; no accounts 
+          or sign-ups required.</p>
+          </div>
+          <ul class="tag">
+            <li class="html">html</li>
+            <li class="cascade">css</li>
+            <li class="javascript">javascript</li>
+          </ul>
+
+          <div class="see-project-one">
+           <button class="see-project">See project</button>
+           </div>
+
+      </section>
+
+      <section class="card-Work-container-2">
+        <div class="snapshot-porfolio">
+          <img src="./images/snap 4.png" alt="">
+        </div>
+        <h2>Multi-Post <br> Stories</h2>
+        <ul class="tonic">
+          <li class="canopy">CANOPY</li>
+          <li class="back-end">Back End Dev</li>
+          <li class="year">2015</li>
+        </ul>
+        <p>A daily selection of privately 
+          personalized reads; no accounts 
+          or sign-ups required.</p>
+          <ul class="tag">
+
+              <li class="html">html</li>
+              <li class="cascade">css</li>
+              <li class="javascript">javascript</li>
+            </ul>
+          <div class="see-project-one">
+           <button class="see-project">See project</button>
+           </div>
+
+      </section>
+      
+      <section class="about-myself">
+        <h3>About <br> Myself</h3>
+        <p>Hello I’m a software developer! I can help you build a product , 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</p>
+          <a href="#" class="connect">LET'S CONNECT</a>
+          <ul class="icons">
+            <li> <img src="./images/twitter.png" alt=""></li>
+            <li> <img src="./images/Linkedin icon.png" alt=""></li>
+            <li> <img src="./images/medium.png" alt=""></li>
+            <li> <img src="./images/github.png" alt=""></li>
+            <li> <img src="./images/angellist.png" alt=""></li>
```

Comment:
Good job implementing the social links, however I think it would be more appropriate to make sure The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag) apart from being sematically correct it would allow us to connect our social links
ex: `<li> <a href="social link"><img src="./images/twitter.png" alt=""></a></li>`